examples/medplum-chat-demo/src/App.tsx

```diff
@@ -0,0 +1,120 @@
+import { formatSearchQuery, getReferenceString, Operator, ProfileResource } from '@medplum/core';
+import { AppShell, Loading, Logo, NotificationIcon, useMedplum, useMedplumProfile } from '@medplum/react';
+import { IconClipboardCheck, IconFileImport, IconMail, IconMessage, IconMessage2Bolt } from '@tabler/icons-react';
+import { Suspense } from 'react';
+import { NavigateFunction, Route, Routes, useNavigate } from 'react-router-dom';
+import { CommunicationPage } from './pages/CommunicationPage';
+import { LandingPage } from './pages/LandingPage';
+import { PatientPage } from './pages/PatientPage';
+import { ResourcePage } from './pages/ResourcePage';
+import { SearchPage } from './pages/SearchPage';
+import { SignInPage } from './pages/SignInPage';
+import { UploadDataPage } from './pages/UploadDataPage';
+
+export function App(): JSX.Element | null {
+  const medplum = useMedplum();
+  const profile = useMedplumProfile();
+  const navigate = useNavigate();
+
+  // Create an array of links and add a link to all active threads in the project
+  const userLinks = [
+    { icon: <IconMessage />, label: 'All Threads', href: '/Communication?part-of:missing=true&status:not=completed' },
+  ];
+
+  // Format a search query to get all active threads assigned to the current user
+  const myThreadsQuery = formatSearchQuery({
+    resourceType: 'Communication',
+    filters: [
+      { code: 'part-of:missing', operator: Operator.EQUALS, value: 'true' },
+      { code: 'recipient', operator: Operator.EQUALS, value: (profile && getReferenceString(profile)) as string },
+      { code: 'status:not', operator: Operator.EQUALS, value: 'completed' },
+    ],
+  });
+
+  // Add this link to the link array
+  userLinks.push({ icon: <IconMessage2Bolt />, label: 'My Threads', href: `/Communication${myThreadsQuery}` });
+
+  if (medplum.isLoading()) {
+    return null;
+  }
+
+  return (
+    <AppShell
+      logo={<Logo size={24} />}
+      menus={[
+        {
+          // A section of the sidebar that displays the links defined above
+          title: 'My Links',
+          links: userLinks,
```

Comment:
My suggestion us to get rid of the `userLinks` variable altogether, and just add the links inline here